lib/utils/timetable.test.ts

```diff
@@ -451,66 +451,61 @@ describe("resolveDevelopmentPlanCSV", () => {
   });
 });
 
-describe("getStageProgress", () => {
-  test("returns IN PROGRESS when startDate is current date", () => {
-    const startDate = new Date();
-
-    const progress = getStageProgress(startDate.toISOString());
-
-    expect(progress).toBe("IN PROGRESS");
-  });
-
-  test("returns NOT STARTED when startDate is in the future", () => {
-    const startDate = new Date();
-
-    startDate.setMonth(startDate.getMonth() + 1);
-
-    const progress = getStageProgress(startDate.toISOString());
-
-    expect(progress).toBe("NOT STARTED");
-  });
-
-  test("returns FINISHED when startDate is in the past and there is no endDate", () => {
-    const startDate = new Date();
-
-    startDate.setMonth(startDate.getMonth() - 1);
-
-    const progress = getStageProgress(startDate.toISOString());
-
-    expect(progress).toBe("FINISHED");
-  });
-
-  test("returns IN PROGRESS when startDate is in the past and endDate is in the future date", () => {
-    const startDate = new Date();
-
-    startDate.setMonth(startDate.getMonth() - 1);
-
-    const endDate = new Date();
-
-    endDate.setMonth(startDate.getMonth() + 1);
-
-    const progress = getStageProgress(
-      startDate.toISOString(),
-      endDate.toISOString()
-    );
-
-    expect(progress).toBe("IN PROGRESS");
-  });
-
-  test("returns FINISHED when endDate date is in the past", () => {
-    const startDate = new Date();
-
-    startDate.setMonth(startDate.getMonth() - 2);
-
-    const endDate = new Date();
-
-    endDate.setMonth(startDate.getMonth() - 2);
+interface StageProgressTestCase {
+  lastUpdatedDate: string;
+  startDate: string;
+  endDate?: string;
+  expectedProgress: string;
```

Comment:
type for expectedProgress could be StageProgress as that is what getStageProgress returns

Reply:
Fair point, I can clean that up